### PR TITLE
documents/README.org: help for running from source with Guix

### PR DESCRIPTION
**Pull request by khinsen**

### CODE REVIEW

**Comment by jmercouris** [comment]
@Ambrevar can you verify? I don't use Guix, so I am not sure.

File: documents/README.org [hunk 1]
```diff
@@ -18,6 +18,18 @@
...
+source so that you can skip the compilation step and iterate faster.
+
+It is most convenient to clone the Next repository into =~/common-lisp= (or [[https://www.common-lisp.net/project/asdf/asdf.html#Configuring-ASDF-to-find-your-systems][another directory where ASDF will find it]]). You can then start Next using the command
+#+begin_src sh
+guix environment next -- sbcl --eval '(require :asdf)' --eval '(asdf:load-system "next")' --eval '(next:start)'
+#+end_src
+You can even start a swank server immediately at startup using
+#+begin_src sh
+guix environment next -- sbcl --eval '(require :asdf)' --eval '(asdf:load-system "next")' --eval '(next:start-swank)' --eval '(next:start)'
+#+end_src
```
Comment (Ambrevar) [inline documents/README.org]: Maybe a new line here.

File: documents/README.org [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
 
+** Run from source
+If you are developing Next, you might prefer to run Next directly from
+source so that you can skip the compilation step and iterate faster.
+
+It is most convenient to clone the Next repository into =~/common-lisp= (or [[https://www.common-lisp.net/project/asdf/asdf.html#Configuring-ASDF-to-find-your-systems][another directory where ASDF will find it]]). You can then start Next using the command
+#+begin_src sh
+guix environment next -- sbcl --eval '(require :asdf)' --eval '(asdf:load-system "next")' --eval '(next:start)'
+#+end_src
+You can even start a swank server immediately at startup using
```
Comment (Ambrevar) [inline documents/README.org]: Swank (capital)

**Comment by Ambrevar** [comment]
Looking good, thanks for working on this!

Can you also fix the typo "readme.ord" in the commit message?  Thanks!

**Comment by khinsen** [comment]
Added a few linebreaks. I use `visual-line-mode` on org files, so the text looks fine on my screen without any newlines.

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**